src/js/nav/globalNav.test.js

```diff
@@ -81,7 +110,7 @@ describe('global nav with API restricted sub items', () => {
     axiosSpy.mockReset();
   });
 
-  test('should display sub item with positive API response', async () => {
+  test.only('should display sub item with positive API response', async () => {
```

Comment:
You should test all tests and not this test only.

Reply:
yeah left it there accidentally.